kube-core/src/admission.rs

```diff
@@ -307,10 +305,7 @@ impl AdmissionResponse {
             },
             uid: Default::default(),
             allowed: false,
-            result: Status {
-                reason: Some(reason.to_string()),
-                ..Default::default()
-            },
+            result: Status::failure("", &reason.to_string()),
```

Comment:
Minor nit (and technically unrelated to the PR), but it would make more sense to have `message` be the field passed by `invalid`, and we can set the shorter `reason` to be `InvalidRequest`.

Reply:
Done